frontend/www/js/omegaup/components/problem/SearchBar.vue

```diff
@@ -4,7 +4,7 @@
       <div class="form-group mr-2" v-if="tags.length !== 0">
         <div class="mr-1" v-for="tag in tags">
           <input type="hidden" name="tag[]" v-bind:value="tag" />
-          <span class="badge badge-secondary m-1 p-2">{{ tag }}</span>
+          <span class="badge badge-secondary m-1 p-2">{{ T[tag] }}</span>
```

Comment:
qué pasa si `tag` no se encuentra en `T`? qué te parece mejor

```suggestion
          <span class="badge badge-secondary m-1 p-2">{{ T[tag] ?? tag }}</span>
```

Reply:
Colocarlo así me marca error, que tal `T[tag] ? T[tag] : tag`